README.md

```diff
@@ -119,7 +118,7 @@ The following settings can be adjusted in preprocessing using a config file (see
     channel number (1 + max existing channel), write tiles in the same directory as the rest of the
     channels, and add the new mask channel to frames metadata. 'as_mask' will write mask tiles in a new directory
     and not add them to metadata.
-    * shape_order (str): 'zyx' (default) or 'yxz'. Order of tile dimensions
+    * image_format (str): 'zyx' (default) or 'yxz'. Order of tile dimensions
```

Comment:
tiling uses image_format and dataset uses shape_order, we should choose one!

Reply:
Definitely - sorry for the mess!

Reply:
sorry, I also didn't check this variable during the tiling PR!